pysilcam/tests/test_process.py

```diff
@@ -8,6 +8,8 @@
 import os
 import unittest
 import pandas as pd
+import shutil
+import glob
```

Comment:
F811: redefinition of unused 'glob' from line 7  <!-- Automatically posted by Lintly -->